src/MonacoContainer/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { type CSSProperties } from "react";
```

Comment:
`import { type ... }` is new for me; I like it, I always used `import type { ... }`.

Let's have consistency here and use only this option. I see in some places `import type { ... }` is used. Your option is better because whenever I want to import something else from the same module/package I can do it in the same import statement weather it's type or not.

Reply:
Cool - fixed it and added lint rules to make sure it stays consistent